watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -374,9 +381,13 @@ def build(bio_P=False):
     )
 
     m.fs.aerobic_reactors = (m.fs.R5, m.fs.R6, m.fs.R7)
-    for R in m.fs.aerobic_reactors:
-        iscale.set_scaling_factor(R.KLa, 1e-2)
-        iscale.set_scaling_factor(R.hydraulic_retention_time[0], 1e-3)
+    if bio_P:
+        for R in m.fs.aerobic_reactors:
+            iscale.set_scaling_factor(R.KLa, 1e-1)
+            iscale.set_scaling_factor(R.hydraulic_retention_time[0], 1e-2)
+    else:
+        for R in m.fs.aerobic_reactors:
+            iscale.set_scaling_factor(R.KLa, 1e-1)
```

Comment:
Might need to revisit if we could stabalize the system with only set scaling factors for KLa

Reply:
Possibly - for now the flowsheet won't solve if we don't scale HRT for bio_P=True